README.md

```diff
@@ -886,10 +1202,10 @@ third-party graphing tool. The result is shown here:
 
 ## Displaying CrateDB data as a Grafana Dashboard
 
-**CrateDB** has been chosen as the time-series data sink, as it integrates
-seamlessly with the [Grafana](https://grafana.com/) time series analytics tool.
-Grafana can be used to display the aggregated sensor data - a full tutorial on
-building dashboards can be found
+**CrateDB** has been chosen as the time-series data sink of preference, as it 
```

Comment:
At the beginning it was tested with other dbs, but the support for others was not continued. So, it was a preferred at some point. Removed "preference" to avoid confusion.